tests/unit_tests/preprocessing_unit_test.py

```diff
@@ -84,10 +86,12 @@ def test_setting_preprocessing_params_from_validation_set(self):
             ],
         }
         trainset = COCODetectionDataset(**train_dataset_params)
-        train_loader = dataloaders.get(dataset=trainset, dataloader_params={"collate_fn": DetectionCollateFN()})
+        assert isinstance(trainset, HasPreprocessingParams)
```

Comment:
Not sure if it makes a difference, but we usually assert with self.XXX
I think isinstance is also supported: you can try `self.assertIsInstance` or something like that